### PR TITLE
Redsys: Update to $0 verify

### PR DESCRIPTION
**Pull request by almalee24**
Remote:
22 tests, 54 assertions, 3 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications 86.3636% passed

Unit:
34 tests, 117 assertions, 0 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications 100% passed

### CODE REVIEW

File: test/remote/gateways/remote_redsys_test.rb [hunk 1]
```diff
@@ -175,21 +175,16 @@ def test_failed_void
...
 
-    void = @gateway.void(authorize.authorization)
-    assert_success void
-
-    another_void = @gateway.void(authorize.authorization)
-    assert_failure another_void
-    assert_equal 'SIS0222 ERROR', another_void.message
+    void = @gateway.void(authorize.authorization << '123')
+    assert_failure void
+    assert_equal 'SIS0007 ERROR', void.message
```
Comment (curiousepic) [inline test/remote/gateways/remote_redsys_test.rb:180]: I think these same changes are needed for the `sha256` version of the remote tessts as well

**Review by curiousepic — CHANGES_REQUESTED** [review]
I think this needs one more test change, but after that should be GTG

**Review by curiousepic — APPROVED** [review]
Thanks - sorry for late re-approval

**Comment by aenand** [comment]
Should we also make this update to Redsys Rest?

**Comment by almalee24** [comment]
> Should we also make this update to Redsys Rest?

Just seeing this but I'll create a separate ticket to update RedsysRest.

**Pull request merged**